public/js/app.js

```diff
@@ -1,10 +1,22 @@
 // app.js
-$(document).on('click', '.load-more-comment', function (event) {
+document.addEventListener('click', function (event) {
+    if (!event.target.matches('.load-more-comment')) return;
     event.preventDefault();
-    $.get($(this).attr('href'), function (data) {
-        $('.load-more-comment-container').hide();
-        $('.comment-feed-container').append(data);
-    });
+
+    var xhr = new XMLHttpRequest();
+    xhr.open('GET', event.target.href);
+    xhr.setRequestHeader('X-Requested-With', 'XMLHttpRequest');
+    xhr.send();
+    xhr.onreadystatechange = function() {
+        if (xhr.readyState === 4) {
+            if(xhr.status === 200) {
+                document.querySelector(".load-more-comment-container").style.display = "none";
+                document.querySelector(".comment-feed-container").insertAdjacentHTML('beforeend', xhr.responseText);
+            } else {
+                document.querySelector(".comment-feed-container").innerHTML += 'An error occurred during your request: ' +  xhr.status + ' ' + xhr.statusText;
```

Comment:
```suggestion
                document.querySelector(".comment-feed-container").innerText += 'An error occurred during your request: ' +  xhr.status + ' ' + xhr.statusText;
```

Reply:
What would be the benefit of treating the HTML we retrieved as text?